activerecord/lib/active_record/connection_adapters/abstract_adapter.rb

```diff
@@ -441,6 +441,10 @@ def supports_concurrent_connections?
         true
       end
 
+      def supports_index_hints?
```

Comment:
There is already `supports_optimizer_hints?` then I'm little bit confused with this name.
How about `supports_use_index_hints?`?

Reply:
Good point, I've updated the method name.